### PR TITLE
responsive

### PR DESCRIPTION
**Pull request by adrijshikhar**
fixed responsiveness for mobile view
added hamburger in navbar for mobile view
added media-query mixins

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -307,12 +338,13 @@ <h1>Stats</h1>
         75% Teetotallers
       </div>
       <div class="entry">
-        40% CS undergrad members vs 60% non CS (all time)
+        40% CS undergrad members vs<br> 60% non CS (all time)
```
Comment (pandeykartikey) [inline index.html]: Why a br?

File: index.html [hunk 2]
```diff
@@ -298,7 +329,7 @@ <h1>Stats</h1>
         3 Dribbble Accounts
       </div>
       <div class="entry">
-        47 Public Repositories on GitHub
+        47 Public Repositories on<br> GitHub
```
Comment (pandeykartikey) [inline index.html]: Why <br>?

File: index.html [hunk 3]
```diff
@@ -226,19 +254,22 @@ <h3>Live</h3>
...
     </div>
   </section>
 
   <section id='fp' class='bg-spl'>
     <div class="info">
       <h3>File Panda</h3>
       <p>
-        File Panda is the intranet repository of free and open source applications. It is a self updating application which ensure that it always has the latest version of each software.
+        File Panda is the intranet repository of free and open source applications. It is a self updating application
+        which ensure that it always has the latest version of each software.
```
Comment (pandeykartikey) [inline index.html]: Why such changes?

File: index.html [hunk 4]
```diff
@@ -50,13 +58,16 @@ <h2>IIT Roorkee, India</h2>
     <div class="content">
       <h1>About SDSLabs</h1>
       <p>
-      SDSLabs is a student group that constantly tries to innovate and foster technical activities in campus. Our members work in C++, Go, PHP, Ruby, Node.js, Python, sometimes even delving into socket level programming.
+        SDSLabs is a student group that constantly tries to innovate and foster technical activities in campus. Our
```
Comment (pandeykartikey) [inline index.html]: Why have you added new lines in the content throughout the page?

File: css/styles.scss [hunk 5]
```diff
@@ -1,297 +1,453 @@
...
 }
 
 section.no-height {
-  min-height: initial;
+	min-height: fit-content;
 }
 
 .clear-height {
-  min-height: initial;
+	min-height: initial;
```
Comment (pandeykartikey) [inline css/styles.scss]: As indentation changes are many so remove them from this PR and make a separate PR for them.

File: index.html (hunk 1 above)
Comment (adrijshikhar) [reply index.html]: to fit this but i have some changes and removed this

File: css/styles.scss (hunk 5 above)
Comment (adrijshikhar) [reply css/styles.scss]: i have enabled auto indentation on save and auto save after every one second in vscode and now how am i supposed to revert the indentation?

Comment (pandeykartikey) [reply css/styles.scss]: cherry-picking commits may help.

File: index.html (hunk 1 above)
Comment (pandeykartikey) [reply index.html]: Remove it then.
